src/shared/store/modeSlice.ts

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const modeSlice = createSlice({
+  name: "mode",
+  initialState: {
+    language: "en",
+  },
+  reducers: {
+    toggleTheme: (state) => {
+      state.language = state.language === "en" ? "ru" : "en";
+    },
+  },
+});
+
+export const { toggleTheme } = modeSlice.actions;
+
+export default modeSlice.reducer;
```

Comment:
why do you need default export? It has some disadvantages and lead to weird bugs sometimes. use specific exports instead